quickstart-mongodbatlas/README.md

```diff
@@ -0,0 +1,35 @@
+quickstart-mongodbatlas
+=============
+## MongoDB Atlas on the AWS Cloud
+
+### Status: early-ALPHA (actively looking for feedback)
+
+![GitHub release (latest by date including pre-releases)](https://img.shields.io/github/v/release/jasonmimick/quickstart-mongodbatlas?include_prereleases&style=for-the-badge)
+
+[![MongoDB Atlas Go Client](https://img.shields.io/badge/Powered%20by%20-go--client--mongodb--atlas-%2313AA52?style=for-the-badge)](https://github.com/mongodb/go-client-mongodb-atlas)
+
+Start here: [beta-launch.md](beta-launch.md)
+
+This Quick Start sets up a flexible, scalable AWS environment for MongoDB Atlas through AWS CloudFormation resources.
+
+|marketing-update-here|MongoDB Atlas is an open source, NoSQL database that provides support for JSON-styled, document-oriented storage systems. 
+Its flexible data model enables you to store data of any structure, and it provides full index support, sharding, and replication.
+
+The Quick Start offers two deployment options:
+
+- Deploying MongoDB Atlas into a new virtual private cloud (VPC) on AWS
+- Deploying MongoDB Atlas into an existing VPC on AWS
+
+You can also use the (sample)[sample] AWS CloudFormation templates as a starting point for your own implementation.
+
+:construction: 
+
+![Quick Start architecture for MongoDB on AWS](atlas-cfn-dataplatform.png)
+
+:construction: 
+
+For architectural details, best practices, step-by-step instructions, and customization options, see the 
+[deployment guide](atlas-cfn-dataplatform.png).
+
+To post feedback, submit feature ideas, or report bugs, use the **Issues** section of this GitHub repo.
```

Comment:
Eventually features will get routed to feedback.mongodb.com and once GA support would be our support teams, fyi.  Issue templates highly recommended to keep issues on point :)

Reply:
added issues & pr tmpl